main.py

```diff
@@ -136,26 +133,30 @@ def validate_year(value):
         "directory", type=str, help="Directory containing weather files"
     )
     parser.add_argument(
-        "-e",
-        "--extremes",
-        type=validate_year,
-        help="Year for which to display extremes",
+
+        "-e", "--extremes", type=validate_year, nargs="+", help="Year(s) for which to display extremes"
+    )
+    parser.add_argument(
+        "-a", "--averages", type=validate_year_month, nargs="+", help="Year/Month(s) for which to display averages"
     )
     parser.add_argument(
-        "-a",
-        "--averages",
-        type=validate_year_month,
-        help="Year/Month for which to display averages",
+        "-c", "--charts", type=validate_year_month, nargs="+", help="Year/Month(s) for which to display temperature bars"
     )
 
     args = parser.parse_args()
-
     weather_man = WeatherMan(args.directory)
 
     if args.extremes:
-        weather_man.process_annual_weather_extremes(args.extremes)
-    elif args.averages:
-        year, month = args.averages.split("/")
-        weather_man.process_monthly_weather_averages(year, month)
-    else:
-        parser.print_help()
+        for year in args.extremes:
+            weather_man.process_annual_weather_extremes(year)
+
+    if args.averages:
+        for year_month in args.averages:
```

Comment:
What's the use of this loop here?

Reply:
While solving merge conflicts, I added this by mistake thinking that the subtask where we had to allow multiple inputs (like weatherman.py   /path/to/files-dir   -c   2011/03   -a   2011/3   -e   2011), was a part of this PR, but it was of the next one actually. I will remove the loop from this code.